tools/export/uvision/__init__.py

```diff
@@ -183,7 +183,7 @@ def format_flags(self):
             flags['c_flags'] + flags['cxx_flags'] + flags['common_flags']
         )
         in_template = set(
-            ["--no_vla", "--cpp", "--c99", "-MMD"] + config_option
+            ["--no_vla", "--cpp", "--cpp11", "--c99", "-MMD"] + config_option
```

Comment:
Are the `--cpp` and `--cpp11` flags exclusive in this case?
```suggestion
            ["--no_vla", "--cpp11", "--c99", "-MMD"] + config_option
```

Reply:
My reading of that was that that was just a list of options that would be eligible for through in the code below. Can you check?

In which case both `--cpp` and `--cpp11` want this behaviour, in case anyone wants to switch profile back to old version.

Reply:
> that was that that

PS Don't report me to tech authors.

Reply:
Oh gosh you're totally right, good eye! In my defense, I have not finished my first cup of coffee this morning 😄